kuksa_viss_client/kuksa_viss_client/__main__.py

```diff
@@ -409,7 +405,19 @@ def do_printTokenDir(self, args):
 
 # Main Function
 def main():
-    clientApp = TestClient()
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--ip', help="VISS Server IP Address", default=DEFAULT_SERVER_ADDR)
```

Comment:
We should not use the term VISS here as it refers to the [W3C Websocket protocol](https://raw.githack.com/w3c/automotive/gh-pages/spec/VISSv2_Transport.html) from the VISS standard. Possibly we could use the term `VSS Server` instead (so that it covers both talking VISS protocol and gRPC)

Same comment applies to all other places - we shall in the long term only use the term VISS if referencing to the protocols specified by W3C in [VISS Core](https://raw.githack.com/w3c/automotive/gh-pages/spec/VISSv2_Core.html) or [VISS Transport](https://raw.githack.com/w3c/automotive/gh-pages/spec/VISSv2_Transport.html) document.

(There are other VISS implementations like a reference implementation at https://github.com/w3c/automotive-viss2 but I do not think that anyone ever has tested the kuksa-viss-client against it)